app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -22,6 +23,9 @@ class Api::V0::UserRolesController < Api::V0::ApiController
     if !is_active.nil?
       active_record = is_active ? active_record.active : active_record.inactive
     end
+    if !is_group_hidden.nil?
+      active_record = active_record.includes(:group).where(group: { is_hidden: is_group_hidden })
+    end
```

Comment:
I feel there is a better way to handle this. Also, I'm not sure if this works as expected in all cases (worked in all cases in my testing though).

Reply:
What kind of better way are you looking for?
You're using a very similar approach for `group_type` below, so I don't see what would be wrong with this approach either

Reply:
As we discussed over slack, I'm convinced that this is a good way. :-)